src/components/buttons/pages/buttons.jsx

```diff
@@ -24,33 +48,32 @@ const Buttons = () => {
   // eslint-disable-next-line react/prop-types
   const getToggleButtons = ({type} = {}) => (
     <>
-      {[...someButtonsWithToggle, 'transparent-peach'].includes(type) && (
+      {[...someButtonsWithToggle].includes(type) && (
         <Button
           type={type}
-          toggle="peach"
           icon={<Icon type="heart" color="adaptive" size={24} />}
           style={{marginBottom: '12px'}}
         >
           {buttonsText}
         </Button>
       )}
-
-      {[...someButtonsWithToggle, 'transparent-mustard'].includes(type) && (
+      {[...someButtonsWithToggle, 'transparent-red'].includes(type) && (
         <Button
           type={type}
-          toggle="mustard"
+          toggle="red"
           icon={<Icon type="heart" color="adaptive" size={24} />}
           style={{marginBottom: '12px'}}
         >
           {buttonsText}
         </Button>
       )}
 
-      {[...someButtonsWithToggle, 'transparent-blue'].includes(type) && (
+      {[...someButtonsWithToggle].includes(type) && (
         <Button
           type={type}
-          toggle="blue"
-          icon={<Icon type="heart" color="adaptive" size={24} />}
+          toggle="yellow"
+          icon={getIcon(type)}
```

Comment:
Could we have the same icons for the toggle buttons? Buttons above have a `heart` icon, `getIcon` returns `heart_outlined`

Reply:
Those icons are set according to figma: https://www.figma.com/file/VRNGY1WOrIAFMU03pvqn1O/%5BDS%5D-Button?node-id=2985%3A226575.

Reply:
I found that yellow toggles have `heart_outlined` and changed them to `heart`